### PR TITLE
Translations - Improve Spanish

### PR DESCRIPTION
**Pull request by regiregi22**
An update of the spanish strings in the addon that lacked translation

**When merged this pull request will:**
It will completely translate the text strings to Spanish,

### IMPORTANT

- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/airway/stringtable.xml [hunk 1]
```diff
@@ -37,6 +37,7 @@
             <English>Probability for an obstructed airway in percentage that gets rolled every:\n- Falling unconscious\n- Removing KingLT / Guedel Tube</English>
             <German>Atemwegsverlegungswahrscheinlichkeit in prozent die berechnet wird sofern:\n- Bewustlos werden\n- Das Entfernen von Larynxtubus / Guedeltubus\n- Abbrechen der Stabile Seitenlage</German>
             <Czech>Pravděpodobnost obstrukce dýchacích cest v procentech, která se vylosuje při každém:\n- Upadnutí do bezvědomí\n- Odstranění KingLT / GuedelTuby</Czech>
+            <Spanish>Probabilidad en porcentaje de que una vía aérea sea obstruida cada vez que:\n- Cae inconsciente \n- Se quita KingLT / Tubo de Guedel</Spanish>
```
Comment (Gilfor) [inline addons/airway/stringtable.xml]:
```suggestion
            <Spanish>Probabilidad en porcentaje de que una vía aérea sea obstruida cada vez que:\n- Cae inconsciente \n- Se retira el tubo laríngeo / Cánula de Guedel</Spanish>
```

File: addons/airway/stringtable.xml [hunk 2]
```diff
@@ -1200,36 +1202,43 @@
             <English>Remove King LT</English>
             <German>Entferne Larynxtubus</German>
             <Czech>Odstranit King LT</Czech>
+            <Spanish>Quitando KingLT</Spanish>
```
Comment (Gilfor) [inline addons/airway/stringtable.xml]:
```suggestion
            <Spanish>Retirando tubo laringeo</Spanish>
```

File: addons/airway/stringtable.xml [hunk 3]
```diff
@@ -1200,36 +1202,43 @@
...
         </Key>
         <Key ID="STR_KAT_Airway_Cancel_Guedeltubus">
             <English>Remove Guedel Tube</English>
             <German>Entferne Guedeltubus</German>
             <Czech>Odstranit Guedel Tubu</Czech>
+            <Spanish>Quitando Tubo de Guedel</Spanish>
```
Comment (Gilfor) [inline addons/airway/stringtable.xml]:
```suggestion
            <Spanish>Retirando Cánula de Guedel</Spanish>
```

File: addons/airway/stringtable.xml [hunk 4]
```diff
@@ -1200,36 +1202,43 @@
...
             <English>Reusable Airway Items</English>
             <German>Wieder Benutzbare Atemwegsobjekte</German>
             <Czech>Znovupoužitelné předměty pro dýchací cesty</Czech>
+            <Spanish>Objetos reutilizables de vías aéreas</Spanish>
         </Key>
         <Key ID="STR_KAT_Airway_Reusable_AirwayItems_DESC">
             <English>Regain the KingLT or Guedel Tube when manually removing them</English>
             <German>Erhalte den Larynxtubus oder Guedeltubus wieder bei manueller Entfernung</German>
             <Czech>Vrácení KingLT nebo Guedel Tuby při jejich manuálním odebrání</Czech>
+            <Spanish>Obtener de nuevo el KingLT o el Tubo de Guedel cuando se quitan manualmente</Spanish>
```
Comment (Gilfor) [inline addons/airway/stringtable.xml]:
```suggestion
            <Spanish>Recuperar el Tubo laríngeo o la Cánula de Guedel cuando se quitan manualmente</Spanish>
```

File: addons/airway/stringtable.xml [hunk 5]
```diff
@@ -1200,36 +1202,43 @@
...
         </Key>
         <Key ID="STR_KAT_Airway_RemoveLarynxtubus_Log">
             <English>%1 removed KingLT</English>
             <German>%1 hat Larynxtubus entfernt</German>
             <Czech>%1 odstranil KingLT</Czech>
+            <Spanish>Quitados %1 KingLT</Spanish>
```
Comment (Gilfor) [inline addons/airway/stringtable.xml]:
```suggestion
            <Spanish>%1 retiró el tubo laringeo</Spanish>
```

File: addons/airway/stringtable.xml [hunk 6]
```diff
@@ -1200,36 +1202,43 @@
...
         </Key>
         <Key ID="STR_KAT_Airway_RemoveGuedeltubus_Log">
             <English>%1 removed Guedel Tube</English>
             <German>%1 hat Guedeltubus entfernt</German>
             <Czech>%1 odstranil Guedel Tubu</Czech>
+            <Spanish>Quitados %1 Tubo de Guedel</Spanish>
```
Comment (Gilfor) [inline addons/airway/stringtable.xml]:
```suggestion
            <Spanish>%1 retiró la Canula de Guedel</Spanish>
```

**Review by Gilfor — COMMENTED** [review]
Some fixes requested by polskifrog

**Comment by MiszczuZPolski** [comment]
@regiregi22 Could you please update your PR according to @Gilfor suggestions?

**Pull request merged**